build.js

```diff
@@ -251,3 +256,33 @@ function _replaceText(file, find, replace, log) {
     });
   });
 }
+
+function _preprocessStyle(directory, basefile, outfile, log) {
+  const wd = path.resolve( "." );
+  return new Promise((resolve, reject) => {
+    process.chdir(directory);
+    fs.readFile(basefile, 'utf8', (err, txt) => {
+      if (err) {
+        reject(err);
+      } else {
+        less.render(txt, function (err, output) {
+          if (err) {
+            reject(err);
+          } else {
+            fs.writeFile(outfile, output.css, 'utf8', err => {
+              if (err) {
+                reject(err);
+              } else {
+                if (log) {
+                  console.log(log);
+                }
+                process.chdir(wd);
```

Comment:
Good question... I was going with the assumption that it's ok thinking that bad things happened if this fails and the whole build is gonna error out.